components/atoms/InputWithIcons.tsx

```diff
@@ -0,0 +1,50 @@
+import { isNil } from 'lodash'
+import React, { useState } from 'react'
+import { useColorScheme } from 'react-native'
+import { Input, type InputProps, XStack } from 'tamagui'
+
+import getColors from '~/constants/Colors'
+
+type Props = {
+  iconRight?: JSX.Element
+  iconLeft?: JSX.Element
+} & InputProps
+
+const InputWithIcons: React.FC<Props> = (props: Props) => {
+  const colors = getColors(useColorScheme())
+  const [isFocused, setIsFocused] = useState<boolean>(false)
+
+  const inputContainerStyle = {
+    backgroundColor: isFocused ? colors.inputBackground : colors.lightMist,
+    borderColor: isFocused ? colors.primaryTeal : colors.lightMist,
+    borderWidth: isFocused ? 1 : 0
+  }
+
+  return (
+    <XStack
+      alignItems="center"
+      borderRadius={10}
+      paddingHorizontal={24}
+      style={inputContainerStyle}
+    >
+      {!isNil(props.iconRight) && props.iconRight}
+
+      <Input
+        {...props}
+        unstyled
+        marginHorizontal={18}
+        paddingVertical={14}
+        fontFamily={'JetBrainsMonoRegular'}
```

Comment:
use hook [`useAppFont()`](https://github.com/NguyenVanViet2004/FE-GlowUp/blob/main/components/templates/HomeTemplate.tsx) for set fontFamily